source/common/http/conn_manager_utility.cc

```diff
@@ -439,6 +439,10 @@ bool ConnectionManagerUtility::maybeNormalizePath(RequestHeaderMap& request_head
     return true; // It's as valid as it is going to get.
   }
   bool is_valid_path = true;
+  config.normalizePath(request_headers);
```

Comment:
Note that path normalization may fail. In this case the `maybeNormalizePath()` returns false and request is rejected with 400. The `config.normalizePath() ` should indicate if normalization has failed.

Reply:
thanks, fixed.